sdk-setup/src/mer-sdk-chroot

```diff
@@ -291,6 +293,11 @@ prepare_user() {
     fi
     echo "$user ALL=NOPASSWD: ALL" > ${sdkroot}/etc/sudoers.d/$user
     chmod 0440 ${sdkroot}/etc/sudoers.d/$user
+
+    # mount ~/.gnupg (default GNUPGHOME) as read only
+    if [[ -d "${sdkroot}${HOMEDIR}/.gnupg" ]]; then
+        mount --bind -o ro $HOMEDIR/.gnupg ${sdkroot}${HOMEDIR}/.gnupg
+    fi
```

Comment:
I would be more specific about our motivation to do this, so let me suggest a small adjustment (moving under the preceding `if` and bind mounting the directory onto itself):

```suggestion

        # Prevent accidental writes to host's GPG home (SDK uses its own GPG home)
        if [[ -d ${sdkroot}${HOMEDIR}/.gnupg ]]; then
            mount --bind -o ro "${sdkroot}${HOMEDIR}/.gnupg"{,}
        fi
    fi

    echo "$user ALL=NOPASSWD: ALL" > ${sdkroot}/etc/sudoers.d/$user
    chmod 0440 ${sdkroot}/etc/sudoers.d/$user
```